### ISSUE TITLE
Ensure app has keyboard focus

### ISSUE THREAD

**Issue by bryankennedy** [scimusmn/stele#58]
In many of our kiosk installations we expect the browser application to have primary keyboard focus. This allows us to link physical buttons to key-presses via a keyboard encoder.

When Stele boots at start (#56) we should ensure that the app has keyboard focus on all OSes. We can handle this outside of Stele via some scripting but it seems like the window focus tools in the Electron API should be able to handle this within Stele itself. This would save us an automation/setup step.

**Comment by azuldev1** [comment]
Had trouble creating a lack of keyboard focus within Stele. Making it difficult to test this behavior. I added:

> `  // Ensure the application window has focus as well as the embedded content
>   // will be called on settings page and when url is switched to home url
>   mainWindow.webContents.on('dom-ready', () => {
>     console.log('DOM READY');
>     mainWindow.focus();
>     mainWindow.webContents.focus();
>   });`

The idea is that maybe within keyboard/keyboard [encoder dependent interactives](https://github.com/scimusmn/race-iat-test) either the Wrapper Application or the embedded web app does not have focus. With something like a touch screen based interactive the initial touch of the screen could be calling the two different layers of the application into focus. I added the event listener above, which should call both layers into focus. 

Used the [IAT App](https://github.com/scimusmn/race-iat-test) to test, the screen saver for this app is listening for .on keydown event,  which means only keyboard interaction will get you passed the screen saver view. Did not have any issues getting past the screen saver on windows or mac. Tested via application launching at startup and ensuring no initial mouse interactions.

**Comment by bryankennedy** [comment]
Thanks for the explanation and the testing. I merged the code and it seems to work on my system as well. I'll close this out for now and we can see how it works on a few of our systems going forward.
